### DISCUSSION TITLE
How to add a specific value to ABI hash calculation.

### DISCUSSION THREAD

**Discussion by recsater** [microsoft/vcpkg#37707] [Q&A]
I want to use the output of a program for ABI hash calculation. For example, I would like to include the value of `lsb_release -rs` in the ABI hash calculation. How can I do that?

**Comment by Osyotr** [comment]
There's https://github.com/microsoft/vcpkg-tool/pull/797 but you have to write your values to a file first.
There's also https://github.com/microsoft/vcpkg-tool/pull/816 but it hasn't been merged yet.

**Reply by recsater** [reply]
thx so much :heart:

**Reply by recsater** [reply]
It seems that VCPKG_HASH_ADDITIONAL_FILES does not work.

I am using an overlay triplet and added the following line to the triplet
```cmake
set(VCPKG_HASH_ADDITIONAL_FILES "${CMAKE_CURRENT_LIST_DIR}/file1.cmake")
```

I also created a file named file1.cmake in the same location as the triplet file.

Despite changing the contents of the file1.cmake file and repeatedly attempting cmake, the ABI hash does not change.

**Reply by Neumann-A** [reply]
If you did not build vcpkg-tool from source it cannot work since the tool with that change merged has not yet been released.
